<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QDialog" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>363</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Menu principal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/assets/images/menu.png</normaloff>:/assets/images/menu.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 251, 111);</string>
  </property>
  <widget class="QPushButton" name="Inicio">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
color: rgb(0, 0, 255);
font-size: 20px;
font-family: &quot;Comic Sans MS&quot;;
font-weight: bold;
background-color:rgb(255, 200, 0)</string>
   </property>
   <property name="text">
    <string>JUGAR</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>90</y>
     <width>375</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button1">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="assets.qrc">
        <normaloff>:/assets/images/zapelingVerde.png</normaloff>:/assets/images/zapelingVerde.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="assets.qrc">
        <normaloff>:/assets/images/zapelingMorado.png</normaloff>:/assets/images/zapelingMorado.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button3">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="assets.qrc">
        <normaloff>:/assets/images/zapelingAmarillo.png</normaloff>:/assets/images/zapelingAmarillo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Salir">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
color: rgb(0, 0, 255);
font-size: 20px;
font-family: &quot;Comic Sans MS&quot;;
font-weight: bold;
background-color:rgb(255, 200, 0)</string>
   </property>
   <property name="text">
    <string>SALIR DEL JUEGO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>21</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>FLAPPYPROJECT</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
